.github/workflows/ci.yml

```diff
@@ -25,3 +25,26 @@ jobs:
       - run: yarn install --frozen-lockfile
       - run: yarn prettier --check .
       - run: yarn run lint
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.4
+          tools: composer
+      - name: Create Acquia CMS project
+        run: composer create-project acquia/drupal-recommended-project acms
+      - name: Copy Drush configuration
+        run: |
+          mkdir $HOME/.drush
+          cp drush.yml $HOME/.drush
+      - name: Install Acquia CMS and start web server
+        run: |
+          ./vendor/bin/acms acms:install acquia_cms_headless --no-interaction
```

Comment:
Looks like this is installing `acquia_cms_headless` without the content model and default content